### ISSUE TITLE
Video alt text and alt text badge are never displayed.

### ISSUE THREAD

**Issue by NotThatGothic** [bluesky-social/social-app#10917]
### Steps to Reproduce

1. Open any video with alt text on Desktop or Android.
2. The figcaption element with an image's alt text is not displayed on videos.
3. Moving to fullscreen, the video's alt text is still missing.
4. Additionally, the alt text badge is missing from videos with alt text.

### What platform(s) does this occur on?

Android, Web (Desktop), Web (Mobile)

### Device Info

Android version 13, Windows 10 Narrator, Google Chrome, Firefox 

### What version of the app are you using?

Build version: 1.124.0.1207; Bundle info: 6efebbd (prod); Bundle date: 26061219; Platform: android; Platform version: 33; Device ID: 8cb1fe74-10c8-4d29-b295-89b837c15333

### Additional Information

Using Android's Talkback screenreader accessibility feature, it is possible to listen to the alt text, however for hearing impaired users the visible alt text can provide necessary additional context. 

The lack of an alt text badge on videos with alt text means that users are unable to know when videos have alt text.

**Comment by surfdude29** [comment]
This was fixed by #10990 :)
